src/settings/RequestPolicy/components/EditSections/GeneralSection/GeneralSection.js

```diff
@@ -59,6 +59,7 @@ class GeneralSection extends React.Component {
 
     return (
       <Accordion
+        data-testid="generalRequestPolicyFormTestId"
```

Comment:
as discussed should remove `TestId` postfix

Reply:
done

Reply:
done